### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by raunakrai**

### CODE REVIEW

File: Assignment 5/SenerioOne.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+try 
+{
+  function CarAccidentByWoman()
```
Comment (Rohit-verma01) [inline Assignment 5/SenerioOne.js:3]: You can define function outside of the try catch block. Then you call function in try catch. This is more suitable

**Review by Rohit-verma01 — APPROVED** [review]
You are not throwing any exceptions or errors in the try block to catch them in the catch block.

File: Assignment 5/SenerioOne.js (hunk 1 above)
Comment (rajeevdey2002) [inline Assignment 5/SenerioOne.js:3]: Please follow a particular type of naming convention

File: Assignment 5/SenerioOne.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+try 
+{
+  function CarAccidentByWoman() 
+  {
+    console.log("Women did accident of her Car");
+  }
```
Comment (RakshitaAgarwal) [inline Assignment 5/SenerioOne.js:6]: please follow the same pattern of formatting rules in each of the files, as the indentation is SenerioOne.js has lesser horizontal indentation from left space as compared to SenerioTwo.js

File: Assignment 5/SenerioOne.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    console.log(
+      "Man is shocked after doing the accident and defending himself"
+    );
+  }
+
+  CarAccidentByWoman();
+  shockedReactionByMan();
+} catch (error) 
+{
+  function BlameOfCarAccident(error)
```
Comment (sarthak-agarwal-intimetec) [inline Assignment 5/SenerioOne.js:19]: Use camel case for function Name

**Review by sarthak-agarwal-intimetec — CHANGES_REQUESTED** [review]

**Pull request merged**